src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml

```diff
@@ -1,27 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <files xmlns:dcterms="http://purl.org/dc/terms/">
-  <file filepath="data/path/to/a/random/video/hubble.mpg">
-    <dcterms:type>http://schema.org/VideoObject</dcterms:type>
-    <dcterms:format>video/mpeg</dcterms:format> <!-- (te bepalen adhv het bestand) -->
-    <dcterms:title>hubble.mpg</dcterms:title>
-    <dcterms:accessRights>RESTRICTED_GROUP</dcterms:accessRights>
-  </file>
   <file filepath="data/path/to/a/random/sound/chicken.mp3">
     <dcterms:type>http://schema.org/AudioObject</dcterms:type>
     <dcterms:format>audio/mpeg</dcterms:format>
     <dcterms:title>our daily wake up call</dcterms:title>
     <dcterms:accessRights>RESTRICTED_GROUP</dcterms:accessRights>
   </file>
-  <file filepath="data/reisverslag/deel01.docx">
-    <dcterms:format>application/vnd.openxmlformats-officedocument.wordprocessingml.document</dcterms:format>
-  </file>
-  <file filepath="data/reisverslag/deel01.txt">
-    <dcterms:format>text/plain</dcterms:format>
-  </file>
-  <file filepath="data/reisverslag/deel02.txt">
-    <dcterms:format>text/plain</dcterms:format>
+  <file filepath="data/path/to/a/random/video/hubble.mpg">
+    <dcterms:type>http://schema.org/VideoObject</dcterms:type>
+    <dcterms:format>video/mpeg</dcterms:format> <!-- (te bepalen adhv het bestand) -->
```

Comment:
No Dutch.

PS: should have placed [yesterdays](https://github.com/DANS-KNAW/easy-split-multi-deposit/pull/58#discussion_r109425668) message here but then I didn't find this part.